backend/pdf/printer.ts

```
@@ -0,0 +1,1289 @@
+    options.yStart -= fontSize * 1.25
+
+    return await this.drawer.drawTable(this.report.days, columns, options)
+  }
+
+  t(textIdentifier: string, interpolation: any = {}) {
+    return i18n.t(textIdentifier, { lng: this.drawer.settings.language, ...interpolation }) as string
+  }
+}
+
+// From https://github.com/Hopding/pdf-lib
+const lineSplit = (text: string) => text.split(/[\n\f\r\u000B]/)
```

Biome (1.9.4)

[error] Unexpected control character in a regular expression.

Control characters are unusual and potentially incorrect inputs, so they are disallowed.

(lint/suspicious/noControlCharactersInRegex)